src/FlaUI.Core/AutomationElements/TextBox.cs

```diff
@@ -38,6 +58,11 @@ public string Text
                 {
                     return textPattern.DocumentRange.GetText(Int32.MaxValue);
                 }
+		string textWin32 = GetTextWin32();
```

Comment:
Indentation seems to be off, please auto format the document before commiting.. There are also sometimes tabs instead of spaces.

Reply:
I replaced tabs with spaces.